src/components/FeedbackModal.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import { FiStar } from 'react-icons/fi';
+import Response from './Response';
+
+const FeedbackModal = () => {
+    const [isOpen, setIsOpen] = useState(true);
+    const [isResponseVisible, setIsResponseVisible] = useState(false);
+    const [responseMessage, setResponseMessage] = useState('');
+    const [responseSuccess, setResponseSuccess] = useState(false);
+    const [rating, setRating] = useState(0);
+    const [hoverRating, setHoverRating] = useState(undefined);
+    const [comment, setComment] = useState('');
+    const wordCount = comment.trim().split(/\s+/).filter(Boolean).length;
+
+    const onStarClick = (index) => {
+        setRating(index);
+    };
+
+    const onStarHover = (index) => {
+        setHoverRating(index);
+    };
+
+    const handleCloseClick = () => {
+        setIsOpen(false);
+    };
+
+    const handleSendClick = () => {
+        if (wordCount >= 10) {
+            setResponseSuccess(true);
+            setResponseMessage("Review successfully submitted");
+            setIsResponseVisible(true);
+            setTimeout(() => setIsResponseVisible(false), 3000); // Hide the response after 3 seconds
+            handleCloseClick();
+        } else {
+            setResponseSuccess(false);
+            setResponseMessage("Minimum word count not reached");
+            setIsResponseVisible(true);
+            setTimeout(() => setIsResponseVisible(false), 3000);
+        }
+    };
+
+    return (
+        <div className="w-full h-full">
+            {isResponseVisible && <Response success={responseSuccess} message={responseMessage} />}
+            {isOpen && (
+                <div className="fixed inset-0 bg-black bg-opacity-50 flex justify-center items-center">
+                    <div className="bg-white p-6 rounded-lg shadow-lg flex flex-col items-center space-y-4 w-1/2 h-2/3">
+                        <button className="self-end" onClick={handleSendClick}>×</button>
```

Comment:
If we could simply cancel the review by clicking x instead of sending it or blocking the user if they didnt type 10+ words, that would make the flow better.

Reply:
Because I figure when the order is completed, the first time the customer revisits his completed order, we will prompt him automatically for the review, so at least we can force them to give some sort of review. If they just click x it is hard to differentiate whether they rated us 0 stars or they just didn't feel like reviewing, so I just force them to enter something to either x out or send it does the same.

Reply:
Alright, that's something that can be quickly debated at the next meeting. Just personally, if a website asks me for a review that I can't quit, it's Alt+F4 haha